manim/mobject/geometry/polygram.py

```diff
@@ -618,36 +618,44 @@ def __init__(
         self.stretch_to_fit_width(width)
         self.stretch_to_fit_height(height)
         v = self.get_vertices()
-        if grid_xstep is not None:
+
+        if grid_xstep or grid_ystep:
             from manim.mobject.geometry.line import Line
 
-            grid_xstep = abs(grid_xstep)
-            count = int(width / grid_xstep)
-            grid = VGroup(
-                *(
-                    Line(
-                        v[1] + i * grid_xstep * RIGHT,
-                        v[1] + i * grid_xstep * RIGHT + height * DOWN,
-                        color=color,
+            line_kwargs = {"color": color}
+            line_kwargs.update(
+                {k: kwargs[k] for k in ["stroke_color", "stroke_width"] if k in kwargs}
+            )
```

Comment:
I think this will not be exhaustive enough because the same problems also appear when using `stroke_opacity`

Would there be a problem with just using `**kwargs` ?

Reply:
I assume #3428 solved this